### ISSUE TITLE
Error in output

### ISSUE THREAD

**Issue by obrejla** [anacierdem/vscode-requirejs#5]
Just another error appeared in console:

```
shell.ts:426 Cannot read property '2' of null: TypeError: Cannot read property '2' of null
    at referenceProvider.provideDefinition (/Users/ondrej/.vscode/extensions/lici.require-js-0.0.16/extension.js:130:45)
    at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:14:6406
    at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:6:17664
    at new n.Class.derive._oncancel (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:6:13768)
    at Object.h [as asWinJsPromise] (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:6:17627)
    at e.provideDefinition (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:14:6360)
    at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:14:16675
    at t._withAdapter (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:14:15295)
    at t.$provideDefinition (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:14:16637)
    at t.e.handle (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:12:20690)
```

**Comment by anacierdem** [comment]
Expect to be fixed with version 0.0.18

**Comment by obrejla** [comment]
Can not reproduce, thanks.
